### PR TITLE
add ideas of Volcano project for community bridge 2022 Q2

### PR DESCRIPTION
**Pull request by Thor-wl**
Signed-off-by: Thor-wl <13164644535@163.com>

### CODE REVIEW

File: lfx-mentorship/2022/02-Summer/project_ideas.md [hunk 1]
```diff
@@ -29,4 +29,18 @@ Project maintainers and mentors, please submit the ideas below (under the Propos
 
 ## Proposed Project ideas
 
+### Volcano
 
+#### Official Website Docs Enhancement
+
+- Description: Official website docs has not been updated for a long time including technology docs, talks, best practice and so on, which bothers users and developers a lot.  
+- Recommended Skills: Kubernetes, golang, English, Chinese
```
Comment (nate-double-u) [inline lfx-mentorship/2022/02-Summer/project_ideas.md:37]: I  suggest that you add "Technical writing" to the list of recommended skills here as well.

[K8s SIG-Network APIs did a similar project this term](https://github.com/cncf/mentoring/pull/454/files), they also asked for writing samples from applicants.

Comment (nate-double-u) [reply lfx-mentorship/2022/02-Summer/project_ideas.md:37]: I should also add, thanks so much for suggesting a non-code project, this is great!

Comment (Thor-wl) [reply lfx-mentorship/2022/02-Summer/project_ideas.md:37]: Thanks for your advice, @nate-double-u . I'm OK about that.

Comment (Thor-wl) [reply lfx-mentorship/2022/02-Summer/project_ideas.md]: I've added the skill to the description. Please take a review. Thanks!

**Comment by nate-double-u** [comment]
Hi @Thor-wl, could you provide me Liang Tang's email address? I need it to add them to the mentorship. (You can reach out here, in Slack, or by email at natew@cncf.io). Thanks!

**Comment by Thor-wl** [comment]
> Hi @Thor-wl, could you provide me Liang Tang's email address? I need it to add them to the mentorship. (You can reach out here, in Slack, or by email at [natew@cncf.io](mailto:natew@cncf.io)). Thanks!

OK. His email is 1074461480@qq.com.

**Pull request merged**